[libobject] Store definition location in the .vo files (#16261)
coq/coq: doc/changelog/13-misc/16261-libobj+locate.rst

- **Added:**

>   Coq now can store common metadata for objects in the .vo files;
>   in particular, we store precise file location for definitions
>   and inductives, which is used by ``Search`` and ``Locate``.
>   Other features such as attaching documentation to commands are
>   also possible

Reviewer: ```suggestion
  Coq now can store common metadata for objects in .vo files;
  in particular, it stores the file location for definitions
  and inductives, which is used by ``Search`` and ``Locate``.
  Developers can easily add additional types of metadata.
```

"precise" seemed unnecessary

I think this is only interesting to developers, not end users.  IIUC adding and accessing metadata requires changing coq.  Presumably there's an internal API to find out what metadata is attached to an object.

>   (`#16261 <https://github.com/coq/coq/pull/16484>`_,

Reviewer: ```suggestion
  (`#16484 <https://github.com/coq/coq/pull/16484>`_,
```

>   (`#16261 <https://github.com/coq/coq/pull/16261>`_,

Reviewer: ```suggestion
  `#16261 <https://github.com/coq/coq/pull/16261>`_,
```
